notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb

```diff
@@ -0,0 +1,761 @@
+{
```

Comment:
<p>go with <code>SDSSr_65mm</code> </p><p>I commented on the jira ticket. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>